Amplify/Categories/Hub/HubChannel.swift

```diff
@@ -15,6 +15,8 @@ public enum HubChannel {
     /// Hub messages relating to Amplify API
     case api
 
+    case auth
```

Comment:
Hm. I agree with you that adding a comment, `/// Hub messages relating to Amplify Auth` doesn't add much value. Can we get rid of all of these comments, here, so we don't have to add one so that `case auth` is consistent?

Reply:
Removed the comments.